llvm/docs/CommandGuide/llvm-objcopy.rst

```diff
@@ -483,6 +483,12 @@ them.
 
  Mark all defined global symbols as weak in the output.
 
+.. option:: --adjust-vma <value>
+  Add ``<value>`` to VMA and LMA address
+
+.. option:: --adjust-section-vma <name>{+|-|=}<value>
+  Adjust section ``<name>`` VMA and LMA address by ``<value>``
```

Comment:
```suggestion
  Adjust section ``<name>`` VMA and LMA address by ``<value>``.
```
I think this could do with some elaboration regarding what `=` means in this context (presumably it means setting it to the value specified, but that isn't "adjust it by something").